contrib/database/sql/driver.go

```diff
@@ -52,18 +53,19 @@ type traceParams struct {
 	meta       map[string]string
 }
 
-func (tp *traceParams) newChildSpanFromContext(ctx context.Context, resource string, query string) *tracer.Span {
+func (tp *traceParams) newChildSpanFromContext(ctx context.Context, resource string, query string) ddtrace.Span {
 	name := fmt.Sprintf("%s.query", tp.driverName)
-	span := tp.config.tracer.NewChildSpanFromContext(name, ctx)
-	span.Type = ext.SQLType
-	span.Service = tp.config.serviceName
-	span.Resource = resource
+	span, _ := tracer.StartSpanFromContext(ctx, name,
+		tracer.SpanType(ext.SQLType),
+		tracer.ServiceName(tp.config.serviceName),
+	)
 	if query != "" {
-		span.Resource = query
-		span.SetMeta(ext.SQLQuery, query)
+		span.SetTag(ext.SQLQuery, query)
```

Comment:
we should remove this one as a follow-up for: https://github.com/DataDog/dd-trace-py/pull/421